silk/models.py

```diff
@@ -54,14 +54,14 @@ def __init__(self, d):
 
 class Request(models.Model):
     id = CharField(max_length=36, default=uuid4, primary_key=True)
-    path = CharField(max_length=190, db_index=True)
+    path = TextField(db_index=True)
```

Comment:
Eh - I'd be more inclined to merge this if we stuck with `CharField` with a max of `255` - Were you able to test `255` for `path` and `view_name` ? - `method` could probably remain the same.